examples/01-Getting-started.ipynb

```diff
@@ -0,0 +1,1012 @@
+{
```

Comment:
<p>I dont think we need to go that detailed in the getting started notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/NVTabular/pull/1633/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
<p>removed</p>